flink-python/lib/LICENSE.cloudpickle

```diff
@@ -1,32 +0,0 @@
-This module was extracted from the `cloud` package, developed by
```

Comment:
I don't think we can delete this file (see comment below)

Reply:
ditto